src/accessibility/rollup.config.js

```diff
@@ -2,14 +2,44 @@ import babel from '@rollup/plugin-babel';
 import nodeResolve from '@rollup/plugin-node-resolve';
 import { terser } from 'rollup-plugin-terser';
 import { rollupPluginDomAccessibilityApi } from '../rollup-plugin-dom-accessibility-api';
+import * as r from 'aria-query';
 
 const extensions = ['.js', '.jsx', '.es6', '.es', '.mjs', '.ts', '.tsx'];
 
+/**
+ * When 'generated:requiredOwnedElements' is imported,
+ * This generates that file from aria-query
+ * Contains a map from role names to required owned elements
+ * @returns {import('rollup').Plugin}
+ */
+const requiredOwnedElementsPlugin = () => ({
+  name: 'required-owned-elements',
+  resolveId: (id) => (id === 'generated:requiredOwnedElements' ? id : null),
+  load(id) {
+    if (id !== 'generated:requiredOwnedElements') return;
+    let output = 'export default new Map([\n';
+    for (const [role, data] of r.roles) {
+      if (data.requiredOwnedElements.length > 0) {
+        // Some of the sub-arrays have two items,
+        // which means that the outer element owns elements
+        // which have a role within another element with another role
+        // Our implementation doesn't support this yet.
```

Comment:
Is there a GH issue we could link to here?

Reply:
I think it would be difficult to come up with a non-contrived example for this. I think we can open an issue if we run into a scenario where this divergence from the spec actually happens, since then we'll have a real-world example. I think it is unlikely for us to run into a scenario where this difference matters.